### ISSUE TITLE
Abnormal behavior - App closes on backpress.

### ISSUE THREAD

**Issue by m-murad** [hawkular/hawkular-android-client#131]
Current behavior - When the user presses backbutton in `Alerts` fragment the app is closed.

Expected behavior - On pressing back button in `Alerts` fragment `Favorites` fragment must appear.

Should I send a PR for this?

**Comment by pilhuhn** [comment]
Sure, sounds good

**Comment by KeenWarrior** [comment]
@free4murad Working on it?

**Comment by m-murad** [comment]
Yes

**Comment by m-murad** [comment]
Can any one please help me with this
![screen shot 2017-03-10 at 10 00 58 pm](https://cloud.githubusercontent.com/assets/17262180/23803642/33f72a84-05dd-11e7-970f-8a2146a3f19f.png)

Not able to understand how I am supposed to do this.

**Comment by danielpassos** [comment]
IMO we should not _fix_ it. It's not a but. If you are on the _Alerts_ and wanna go back to the _Favorites_ you should use the menu. I don't agree to have a queue of fragment in MainActivity. 

TL;DR If you are in the _MainActivity_ and press back button app should be closed. That is the default behavior

**Comment by m-murad** [comment]
@danielpassos 
I Haven't seen this behavior anywhere. 
Apps like `Gmail`, `Play Store`, `Medium`, `utorrent` are using the approach that I have suggested. 

Anyway if you think that we should keep it that way then let it be.

**Comment by danielpassos** [comment]
I see, it's going back to home when press back button, I still don't like it, but let's move one with it.
